src/Component/Pokemon.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import {useEffect, useState} from "react";
+
+
+export default function Pokemon({url}) {
+    const [pokemon, setPokemon] = useState('');
+
+    useEffect(() => {
+        async function fetchPokemon() {
+            try {
+                const response = await axios.get(url);
+                // console.log(response.data.name)
+                // console.log(response.data);
+                setPokemon(response.data)
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        if (url) {
+            void fetchPokemon();
+        }
+    }, [url]);
+
```

Comment:
Als je hier
```suggestion
    if (!pokemon) return null;
```
doet, dan kun je de check die hieronder staat weghalen en ben je wat nesting kwijt